yodeploy/test_integration/helpers.py

```diff
@@ -58,15 +62,17 @@ def build_sample(app_name, version='1'):
         'PYTHONPATH': ':'.join(sys.path),
         'BUILD_NUMBER': version,
         'GIT_BRANCH': 'master',
-        'YOLA_SRC': os.path.join(tests_dir, 'samples')
+        'YOLA_SRC': os.path.join(tests_dir, 'samples'),
+        'PYTHONIOENCODING': 'utf-8'
```

Comment:
Okay, that helps, I understand where you're coming from a bit more. I still want to pair with you on it though. I'd still like to pair with you on it though... 😍